app/src/main/java/net/squanchy/navigation/HomeActivity.java

```diff
@@ -92,6 +96,24 @@ private void setupBottomNavigation(InterceptingBottomNavigationView bottomNaviga
         );
     }
 
+    private BottomNavigationSection getSelectedSectionOrDefault(Optional<Bundle> savedInstanceState) {
+        int selectedPageIndex = savedInstanceState.or(new Bundle())
+                .getInt(STATE_KEY_SELECTED_PAGE_INDEX, BottomNavigationSection.SCHEDULE.ordinal());
+        return BottomNavigationSection.values()[selectedPageIndex];
+    }
+
+    private void selectInitialPage(BottomNavigationSection section) {
```

Comment:
It's the same comment you gave last time, I will give you the same answer: there's very little than can be shared out in a single method (order matters and the non-shared stuff is in the middle) so it's not really worth it, as it would require extracting two-three methods of one or two lines.